vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/main/java/com/vaadin/flow/component/combobox/test/ComboBoxPage.java

```diff
@@ -199,6 +200,20 @@ private void createWithUpdatableValue() {
         add(combo, message, button);
     }
 
+    private void createWithUpdateOnValueChange() {
```

Comment:
nit: let's avoid adding new components on these IT pages / old demo pages. Having to render unnecessary component instances makes the tests run slower and difficult to debug. Ideally, one page would contain just 1 component instance. Not a blocker for this PR.